torch/__init__.py

```diff
@@ -1767,6 +1767,10 @@ def __eq__(self, other):
     def __call__(self, model_, inputs_):
         return self.compiler_fn(model_, inputs_, **self.kwargs)
 
+    def reset(self):
+        if hasattr(self.compiler_fn, "reset"):
```

Comment:
Do you really want to swallow this call entirely if there is no reset on compiler_fn? Seems better to error no?

Reply:
Not all backend compilers define reset. I dont think its better to error.

Reply:
Are you unconditionally calling reset somewhere?